spec.html

```diff
@@ -31027,9 +31036,9 @@ <h1>String.prototype.split ( _separator_, _limit_ )</h1>
           1. Let _p_ be 0.
           1. Let _q_ be _p_.
           1. Repeat, while _q_ &ne; _s_,
-            1. Let _e_ be SplitMatch(_S_, _q_, _R_).
-            1. If _e_ is *false*, set _q_ to _q_ + 1.
+            1. If ! StringIncludesAt(_S_, _R_, _q_) is *false*, then set _q_ to _q_ + 1.
             1. Else,
+              1. Let _e_ be _q_ + the number of code units in _R_.
```

Comment:
This seems just unnecessarily verbose.

```suggestion
              1. Let _e_ be _q_ + the length of _R_.
```